api/kv/doc.go

```diff
@@ -17,4 +17,4 @@
 //
```

Comment:
The copied documentation above this line is no longer correct.  This package doesn't provide "values, numbers[, or] span contexts".